### DISCUSSION TITLE
Get the name of the current tenant

### DISCUSSION THREAD

**Discussion by mattobox** [OrchardCMS/OrchardCore#12126] [Q&A] [answered]
Is there a simple way to get the name of the active tenant? In a decoupled mode.

**Comment by hishamco** [comment, marked as answer]
If you mean the current tenant you could use `ShellSettings.Name`

**Reply by mattobox** [reply]
Yes, this is what I was looking for. Thanks!

**Reply by hishamco** [reply]
Don't forgot to inject `ShellSettings` service

**Reply by hishamco** [reply]
Can you show how you register your service?

**Reply by mattyopp** [reply]
I was trying to register the service in the Program.cs file but it worked once I loaded it within a module. Thanks for pointing me in the right direction.

**Reply by hishamco** [reply]
Try to register it in `Startup` for your module

**Reply by mattobox** [reply]
Yes, that’s the ticket! Thanks
